python/py-graph-tool/Portfile

```diff
@@ -1,39 +1,43 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; c-basic-offset: 4 -*-
 
 PortSystem          1.0
-PortGroup           python 1.0
 PortGroup           active_variants 1.1
+PortGroup           boost 1.0
+PortGroup           python 1.0
 
-set realname        graph-tool
-name                py-${realname}
-version             2.33
-revision            1
+name                py-graph-tool
+version             2.37
+revision            0
 epoch               20190711
+
 categories          python science
 platforms           darwin
 license             LGPL-3
 maintainers         nomaintainer
-description         Efficient python graph module
-long_description    graph-tool is an efficient python module for manipulation \
-                    and statistical analysis of graphs. The internal data \
-                    structures and most algorithms are implemented in C++ with \
-                    the Boost Graph Library.
+
+description         ${python.rootname} is an efficient Python module for \
+                    manipulation and statistical analysis of graphs
+
+long_description    *${description}. The internal data structures and \
```

Comment:
shouldn't the asterisk be in "{}"?  I doubt though that it's needed at all here.

Reply:
Yes, thank you!